### PR TITLE
Fixes #12 and #14 .

### LINKED ISSUE #12: Docs - FigResponseCookies::forget vs FigResponseCookies::expire
Docs added in #11 reference a method called `forget`, the code in #10 adds method called `expire`.

As far as I can see there is no `forget` method, and docs need updating to use `expire`.

### PR DESCRIPTION
**Pull request by harikt**
Correct the doc to use expire method and not forget .

### CODE REVIEW

**Comment by simensen** [comment]
Thanks @harikt

File: README.md [hunk 1]
```diff
@@ -263,13 +263,13 @@ $response = FigResponseCookies::remove($response, 'theme');
 
 #### Forget a Response Cookie
```
Comment (franzliedke) [inline README.md]: You might wanna change this to "Expire", too.

Comment (harikt) [inline README.md]: I thought of changing, but forget makes more sense.

Comment (franzliedke) [inline README.md]: It's more confusing when the method is called `expire`, though. ;)

**Pull request merged**